### PR TITLE
feat: rename GPS to GNSS

### LINKED ISSUE #351: Rename GPS to GNSS
nRF9160 supports mutliple satelite position systems, so use the more generic term Global Navigation Satellite System (GNSS).

See https://github.com/nrfconnect/sdk-nrf/pull/6231

### PR DESCRIPTION
**Pull request by coderbyheart**
nRF9160 supports mutliple satelite position systems, so use the
more generic term Global Navigation Satellite System (GNSS).

Fixes #351

BREAKING CHANGE: this renames the configuration and data property

See https://github.com/nrfconnect/sdk-nrf/pull/6392

### CODE REVIEW

File: docs/aws/desired-cfg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "cfg": {
-    "gpst": 720,
+    "gnst": 720,
```
Comment (peknis) [inline docs/aws/desired-cfg.json]: gnsst?

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/aws/desired-cfg.json (hunk 1 above)
Comment (coderbyheart) [inline docs/aws/desired-cfg.json]:
```suggestion
    "gnsst": 720,
```

**Review by wiba-nordic — APPROVED** [review]

**Review by peknis — APPROVED** [review]

File: docs/devices/TroubleShooting.rst [hunk 2]
```diff
@@ -32,21 +32,21 @@ Since 5G support is currently being deployed by mobile phone network operators,
...
 Try multiple SIM cards from different vendors if you experience connectivity issues.
 
-GPS
-***
+GNSS
+****
 
-Error: GPS_EVT_OPERATION_BLOCKED
-================================
+Error: GNSS_EVT_OPERATION_BLOCKED
```
Comment (simensrostad) [inline docs/devices/TroubleShooting.rst]:
```suggestion
Error: NRF_MODEM_GNSS_EVT_BLOCKED
```

Comment (simensrostad) [reply docs/devices/TroubleShooting.rst]: This event name changed when the GPS module was updated to use the GNSS API in
https://github.com/simensrostad/fw-nrfconnect-nrf/commit/e8488c9d4eb98d7fb877a00bb5043792ea66ed84

**Review by simensrostad — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**